src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java

```diff
@@ -55,6 +55,7 @@ private void processCaseCreatedEvent(CollectionCase collectionCase) {
     newCase.setCaseId(UUID.fromString(collectionCase.getId()));
     setCaseDetails(collectionCase, newCase);
     newCase.setReceiptReceived(false);
+    newCase.setCaseType(collectionCase.getCaseType());
```

Comment:
you don't need to do this... it's already done in `setCaseDetails`

Reply:
Removed.